src/modules/EventuateClient.js

```diff
@@ -611,6 +623,25 @@ export default class EventuateClient {
     }
   }
 
+  parseEvent(eventStr) {
```

Comment:
why do we need it async, @dartvandru? up there, in L367, you can map the `body`, and do `this.decrypt()` and so on